sms-sending/javascript/index.html

```diff
@@ -0,0 +1,101 @@
+<html>
+<head>
+  
+  <title>Auto Send SMS</title>
+  
+  <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>  
+  <script src="https://sdk-cdn.mypurecloud.com/javascript/49.0.0/purecloud-platform-client-v2.min.js"></script>
+ 
+</head>
+
+<body>
+  
+  <div>
+      
+    <label id="state">Status: </label>   
+    <label id="stateVal"></label>         
+  
+  </div>
+
+<script type="text/javascript">
+
+  // Set purecloud objects
+  const platformClient = require('platformClient');
+  const client = platformClient.ApiClient.instance;
+  const conversationsApi = new platformClient.ConversationsApi();
+  
+  // Set variables
+  let webSocket = new WebSocket('wss://carrier-pigeon.us-east-1.mypurecloud.com/v1/channels/btmfetbcc9r489fbrsj918876m');
+  let redirectUri = 'http://localhost:8080/';
+  let clientId = 'babbc081-0761-4f16-8f56-071aa402ebcb';
+  let queueId = '4d3af653-b60c-42f1-b373-7cf2c99610fc';
+  let receivedMsg, customerAddress, conversationId, communicationId, callState, usingMobilePhone;
+  let dropOffMessage = 'Message sent upon dropping off the call.';
+
+  // jQuery call this function when DOM's ready
+  $(document).ready(function() {
+    client.loginImplicitGrant(clientId, redirectUri)
+      .then(() => {
+        console.log('Loged in');
+        document.getElementById('stateVal').innerHTML = 'No incoming call at the moment.'
+        webSocket.onmessage = dropOffsms;
```

Comment:
In addition to showing the code to create a channel and subscribe to topics, there should be a basic check on the topic name coming from these events to discard unwanted messages. At a minimum, you'll get heartbeat messages in addition to the topics you subscribe to.

Reply:
Im thinking of creating a pseudo code on the websocket and direct the readers to a tutorial dedicated to web channel creation. Would that be alright? Or is it better to create the channel and subscription here too?

Reply:
The tutorial code needs to have all the code it needs for someone to only have to plug in authentication details and be able to run it without other modifications. The tutorial needs to create the notification channel and then use the newly created channel to connect the websocket and subscribe to topics. The tutorial doesn't need to go into depth about it, but should link to https://developer.mypurecloud.com/api/rest/v2/presence/understanding_presence.html. 

There's an example of the process to use notifications you can work from here: https://developer.mypurecloud.com/api/tutorials/presence/?language=javascript&step=3